test/metabase/driver/sql_jdbc/sync/describe_table_test.clj

```diff
@@ -152,6 +153,15 @@
                #'sql-jdbc.describe-table/describe-json-xform
                #'sql-jdbc.describe-table/describe-json-rf [json-map]))))))
 
+(deftest get-table-pks-test
+  (mt/test-drivers (mt/sql-jdbc-drivers)
```

Comment:
Not loving that the tests for composite PK case are in a different namespace to the tests for the non-composite PK case.

At the least, perhaps we should add a comment linking them here?

Reply:
yea I don't like that either but it'll need some refactoring for the create dataset code to handle table-def with PKs, so I'm using raw SQL directly.

and yes, I'll add some comment for linking